server/controllers/retailerController.js

```diff
@@ -1,4 +1,5 @@
 const { Retailer } = require("../models");
+const passport = require("passport");
 
 const retailerController = {
   getAuthRetailer({ params }, res) {
```

Comment:
I don't think so? Not really sure what the getAuthRetailer is for. It doesn't get touched when registration/login happens.

Reply:
the getAuthRetailer was for in case the user refreshes their app while logged in I think.  They'd have to get their data based on their authorization token, no?  I'm not 100% sure tho.

Reply:
I don't think we need we'll need this getAuthRetailer route if we do the localStorage method I mentioned below, but idk though, let me know what you think.

Reply:
Okay, Kevin was saying the same thing as you, but I'm still not convinced we don't need it.  Tell you what, if you show me how we don't need it, I'll buy you dinner! haha!